account_analytic_project/model/account_analytic_account.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""Extend account.analytic.account."""
+##############################################################################
+#
+#    Copyright (C) 2015 Therp BV <http://therp.nl>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import _, api, models, fields
+
+
+class AccountAnalyticAccount(models.Model):
+    """Extend account.analytic.account."""
```

Comment:
@NL66278 please remove this docstring too. It does not bring any useful information, and it has been agreed to remove that check from pylint.